snapshot/git/gitdb/schemes.go

```diff
@@ -0,0 +1,65 @@
+package gitdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// GitDB uses several different Schemes to identify, upload and download Values.
```

Comment:
OK; let's take this to the design doc (to avoid similar conversations happening in too many places)?